Tests/Categories/NSString_DPLQuerySpec.m

```diff
@@ -73,7 +73,16 @@
     it(@"should serialize multiple arrays from dictionary into the query string", ^{
         NSDictionary *params = @{ @"beers": @[ @"stout", @"ale" ], @"liquors": @[ @"vodka", @"whiskey" ] };
         NSString *query = [NSString DPL_queryStringWithParameters:params];
-        expect(query).to.equal(@"beers[]=stout&beers[]=ale&liquors[]=vodka&liquors[]=whiskey");
+
+        NSString *queryStringToMatch;
+        NSString *queryStringPartBeers = @"beers[]=stout&beers[]=ale";
+        NSString *queryStringPartLiquors = @"liquors[]=vodka&liquors[]=whiskey";
+        if ([[params allKeys][0] isEqualToString:@"beers"]) {
```

Comment:
@wessmith Just making sure I understand that correctly: you are talking here about ```DPLMutableDeepLink``` not ```DPLDeepLink```right? (```DPLDeepLink``` returns previously set URL without any serialization work).